WeakAurasOptions/RegionOptions/StopMotion.lua

```diff
@@ -384,6 +419,10 @@ local function modifyThumbnail(parent, region, data, fullModify, size)
         region.endFrame = floor( (data.endPercent or 1) * lastFrame) + 1;
         region.foregroundRows = data.customForegroundRows;
         region.foregroundColumns = data.customForegroundColumns;
+        region.foregroundFileWidth = data.customForegroundFileWidth
```

Comment:
There's special code a few lines above this which auto-detects the custom settings from the fileName, which sounds like something that would help your workflow.